components/ws-manager-api/typescript/src/client-provider.ts

```diff
@@ -92,14 +92,14 @@ export class WorkspaceManagerClientProvider implements Disposable {
         let client = this.connectionCache.get(name);
         if (!client) {
             const info = await getConnectionInfo();
-            client = this.createClient(info, grpcOptions);
+            client = client = this.createConnection(WorkspaceManagerClient, info, grpcOptions);
```

Comment:
This must be unintentional, right?

```suggestion
            client = this.createConnection(WorkspaceManagerClient, info, grpcOptions);
```

Reply:
@csweichel this is still not addressed.